### PR TITLE
made first version of solution

### PR DESCRIPTION
**Pull request by Sashk52**

### CODE REVIEW

File: src/main/java/com/dev/movietheatre/dao/impl/OrderDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        }
+    }
+
+    @Override
+    public List<Order> getOrdersHistory(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery(
+                    "from Order o "
+                            + " left join fetch o.tickets "
+                            + " where o.user = :user", Order.class)
```
Comment (mfedenko09) [inline src/main/java/com/dev/movietheatre/dao/impl/OrderDaoImpl.java]: https://mateacademy.slack.com/archives/G01EHB8CWAE/p1612455330236200
```suggestion
                    "from Order o "
                            + " left join fetch o.tickets "
                            + " where o.user = :user", Order.class)
```

File: src/main/java/com/dev/movietheatre/service/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order();
+        order.setTickets(new ArrayList<>(shoppingCart.getTickets()));
+        order.setOrderDate(LocalDateTime.now());
+        order.setUser(shoppingCart.getUser());
+        orderDao.add(order);
+        shoppingCartService.clear(shoppingCart);
+        return order;
```
Comment (mfedenko09) [inline src/main/java/com/dev/movietheatre/service/impl/OrderServiceImpl.java]:
```suggestion
        shoppingCartService.clear(shoppingCart);
        return orderDao.add(order);
```

**Review by mfedenko09 — APPROVED** [review]

File: src/main/java/com/dev/movietheatre/service/impl/OrderServiceImpl.java (hunk 2 above)
Comment (Sashk52) [reply src/main/java/com/dev/movietheatre/service/impl/OrderServiceImpl.java]: done

File: src/main/java/com/dev/movietheatre/dao/impl/OrderDaoImpl.java (hunk 1 above)
Comment (Sashk52) [reply src/main/java/com/dev/movietheatre/dao/impl/OrderDaoImpl.java]: added distinct, done

**Review by mfedenko09 — APPROVED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**